telegram/inline/inputlocationmessagecontent.py

```diff
@@ -43,3 +43,5 @@ def __init__(self, latitude, longitude, live_period=None, **kwargs):
         self.latitude = latitude
         self.longitude = longitude
         self.live_period = live_period
+
+        self._id_attrs = (self.latitude, self.longitude, self.live_period)
```

Comment:
As keeping live locations up to dateis not easy as far as I understood from the recent discussion in the user group, one could arge that one could skip `live_period` here